### ISSUE TITLE
Wifi not working

### ISSUE THREAD

**Issue by LevelAdditional79** [tlefko/Matebook-X-Pro-2018-Ventura#40]
Hello, 

I've installed Monterey version 12.6.2 on my Matebook x pro 2018. I've used your repo version 2.1.8 for this. 

Everything works except for the WiFi connection. Do you have an idea on how to fix it? Because I thought the issue with Intel WiFi cards had been fixed... (my WiFi card is not recognised btw)

- Do I need to update kext or SSDT?
- Do I need to replace the EFI with the EFI of a different release?
- Do I need to downgrade my macOS version?

Your expertise on this matter is appreciated.

Thanks

**Comment by tlefko** [comment]
@LevelAdditional79 You need to use the release from openintelwireless (ITLWM) for Monterey. Sometimes it's glitchy, and the current version is for only Ventura. From the releases, just select Monterey from the dropdown.

**Comment by LevelAdditional79** [comment]
Fixed. Thx bro!

**Comment by Paxolo** [comment]
Hi @tlefko,

I had some spare time so I updated my machine to macOS Ventura using the latest release of your EFI.
Unfortunately, I am experiencing issues with WiFi to the point where the laptop is almost unusable.
I tried looking for an updated version of Airportitlwm, and tried using its newest (unstable) release, with no luck whatsoever.

At the moment of writing this:

- connecting to any WiFi network takes at least 2 minutes
- the laptop randomly loses connection and getting back online is a pain

If anyone knows a way I could fix this, please let me know as it would make the installation almost impeccable!
Thanks

**Comment by tlefko** [comment]
@Paxolo Give the version I just published a shot, let me know if it is still present!

**Comment by Paxolo** [comment]
@tlefko I went to the releases page only to find out that there is no EFI folder available, just the source files! Did I miss something or has it been released elsewhere? I could try and make one by myself alternatively, but if anything goes wrong we'll never know what I have messed up 😂

**Comment by tlefko** [comment]
@Paxolo yep I just realized haha I forgot to attach it 😂 definitely don't compile from source I haven't updated that stuff in forever lol!

**Comment by Paxolo** [comment]
@tlefko the new release increased Wi-Fi performance by a LOT!
It still takes a while to connect to the network, but once it does, I feel like it's much more reliable and the transfer speeds are indeed much higher, so that's nice to see!
Unfortunately, though, trackpad, keyboard and touchscreen are broken for me, as they were on release 1.2.1.
While I was able to get trackpad, keyboard and touchscreen working using the fix provided under issue #33 by GaryYa0 on that, I'm afraid I'll break something on 1.25...

**Comment by tlefko** [comment]
hey, tbh I wouldn’t worry about @Paxolo. Apply the fix and let me know what happens. If it doesn’t work, I will help troubleshoot. Glad the wifi is a bit better! I did my best to fix it up but it’s a little out of control haha

**Comment by Paxolo** [comment]
I tried applying that fix and everything seems to be just fine: trackpad, keyboard and even the touchscreen now work again with no problem! 
It would be nice to see that fix integrated into the next release, as it would make the update process much quicker without the need of external peripherals.
Thank you again for your time @tlefko!

**Comment by tlefko** [comment]
Hey, I am slightly confused as what the fix is exactly. Do you mind sharing
the files you replaced? Honestly, I have not been able to replicate these
peripherals not working on my end so I haven’t been able to test the fix. I
have heard that my patches are specific to my model though, so it would
make sense as to some of it not working. These fixes are much better and
universal, I just got lost reading the thread.

**Comment by tlefko** [comment]
hahaha no worries man that’s really helpful. Did you ever attempt to just
use his ACPI patched with my config plist? If not, that would be super
helpful to know if that was successful. Not sure there are any significant
changes in the config as opposed to the original, just want to see for
testing purposes.

**Comment by Paxolo** [comment]
I did just now and unfortunately the laptop no longer boots to macOS due to what I believe to be ACPI errors. Here's what I see when boot gets stuck:

![image](https://user-images.githubusercontent.com/53195387/229002743-31828689-842b-4bad-87f0-4641d510b688.jpeg)

**Comment by tlefko** [comment]
ok so just from this I can see that the trackpad wouldn’t of worked anyway
even if it even if did boot, So I better take a look at that config, test
it out, and you know I’ll probably just update his config to match the new
updates I changed along with a ACPI files. If it’s not broken, no really
point in to fix it so I’ll just test out his stuff and implement it. You
can use another computer to put back the original EFI/config off a usb to
boot and then replace it on your hard drive, didn’t mean to brick it for
you.

Thanks again for trying this out @Paxolo

**Comment by Paxolo** [comment]
No biggie. I'll do it as soon as I wake up as it's 4am here 😂
Let me know if there's anything else you need me to test, I'm more than happy to help as much as possible!

**Comment by tlefko** [comment]
> No biggie. I'll do it as soon as I wake up as it's 4am here 😂
> Let me know if there's anything else you need me to test, I'm more than happy to help as much as possible!

enjoy the credits list :)

**Comment by tlefko** [comment]
@Paxolo give this a shot. I have integrated everything. Latest Release
